tests/processors/base/normalizers/test_phase_normalizer.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from processors.base.normalizers import get_normalized_phase
+import pytest
+
+#Tests
+class TestValidateRemoteURL(object):
+    @pytest.mark.parametrize('test_input,expected', [ 
```

Comment:
No need to test every possibility here. Try just a couple cases and the "extremes" (e.g. `None`, a phase we don't have the mapping for, etc.)